<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlaxaFlow - Panel de Administraci贸n</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FAFAFA; font-family: 'Segoe UI';</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_header">
      <property name="styleSheet">
       <string notr="true">background-color: white; border-bottom: 2px solid #D7CCC8;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="styleSheet">
          <string notr="true">font-size: 20px; font-weight: bold; color: #3E2723;</string>
         </property>
         <property name="text">
          <string>锔 Administraci贸n del Sistema</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_back">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #795548; color: white; padding: 8px 15px; border-radius: 4px; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Volver al Tablero</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane { border: 1px solid #D7CCC8; } QTabBar::tab { background: #EFEBE9; padding: 10px 20px; color: #5D4037; } QTabBar::tab:selected { background: #FFFFFF; font-weight: bold; border-top: 3px solid #FF9800; }</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_users">
       <attribute name="title">
        <string> Gesti贸n de Usuarios</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_add_user">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #4CAF50; color: white; padding: 6px 12px; border-radius: 4px; font-weight: bold;</string>
            </property>
            <property name="text">
             <string>+ Nuevo Empleado</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_refresh_users">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string> Recargar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_users">
          <property name="styleSheet">
           <string notr="true">background: white; border: 1px solid #ddd; color: #333;</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ID (UUID)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nombre</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Apellidos</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Departamento</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Rol</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_boards">
       <attribute name="title">
        <string> Gesti贸n de Tableros</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_add_board">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #FF9800; color: white; padding: 6px 12px; border-radius: 4px; font-weight: bold;</string>
            </property>
            <property name="text">
             <string>+ Nuevo Tablero</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_refresh_boards">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string> Recargar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_boards">
          <property name="styleSheet">
           <string notr="true">background: white; border: 1px solid #ddd; color: #333;</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ID Tablero</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>T铆tulo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Descripci贸n</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Creado Por (ID)</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
